### PR TITLE
Updated: Keyboardio/atreus

### PR DESCRIPTION
**Pull request by evantravers**
This is basically a replacement for #8492 because I assume @algernon is busy with the keyboardio launch.

The bulk of the work is @algernon's excellent work, I just implemented @Erovia's requests.

### CODE REVIEW

**Comment by timbedard** [comment]
Seems to be working for me!

**Review by jackhumbert — APPROVED** [review]

File: keyboards/keyboardio/atreus/rules.mk [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (noroadsleft) [inline keyboards/keyboardio/atreus/rules.mk:11]:
```suggestion
```

File: keyboards/keyboardio/atreus/rules.mk [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```
Comment (noroadsleft) [inline keyboards/keyboardio/atreus/rules.mk:17]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```

File: keyboards/keyboardio/atreus/rules.mk [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
```
Comment (noroadsleft) [inline keyboards/keyboardio/atreus/rules.mk:26]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
```

File: keyboards/keyboardio/atreus/rules.mk [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = yes        # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no     		# Enable support for HD44780 based LCDs
```
Comment (noroadsleft) [inline keyboards/keyboardio/atreus/rules.mk:33]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output

UNICODE_ENABLE = yes        # Unicode
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by evantravers** [comment]
Thank you @noroadsleft!

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Comment by Xyverz** [comment]
Thank you folks!

**Pull request merged**